plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
I want to play around with this a bit especialyl with @docteurklein suggestion. 

I've never been a big fan of the current opening setup but it is so explicitly #doingitright from the plugin handbook that deviating feels wrong for the spirit of this boilerplate.